content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -374,7 +374,7 @@ Here, `W` is the number of work items.
 ### Specifying your own Pod selector
 
 Normally, when you create a Job object, you do not specify `.spec.selector`.
-The system defaulting logic adds this field when the Job is created.
+The system's default logic adds this field when the Job is created.
```

Comment:
i am not sure about this one it seems like this could be meant by it.

Reply:
The original is a bit awkward. I think the rewording loses the sense of “default” as a verb though.

Reply:
I agree that this sentence is awkward. What about:
Kubernetes automatically adds a selector when the Job is created.